lib/editor/components/ScheduleExceptionForm.js

```diff
@@ -84,6 +84,21 @@ export default class ScheduleExceptionForm extends Component<Props> {
     })
   }
 
+  _renderExceptionExemplars (exemplar: string, value: number): string {
+    let exemplarText = ''
```

Comment:
You can probably delete this variable and just return your string directly and then remove the `break`

Reply:
Good call, changed that in https://github.com/ibi-group/datatools-ui/pull/810/commits/374c73fe49c7ff83f0d27355cc98a4d950f2bf71